### ISSUE TITLE
KUVO it does not work

### ISSUE THREAD

**Issue by LuigiMIX** [erikrichardlarson/unbox#90]
Help me, I write my kuvo id, it doesn't show the playlist, while on the kuvo website it does create the playlist, I work with rekordbox 5.8.7. Thank you

![Captura](https://user-images.githubusercontent.com/112134021/186784775-d40dff59-3d30-4c66-9e6b-5d5ff12e9277.PNG)
![Captura2](https://user-images.githubusercontent.com/112134021/186784779-9c18e27a-4a9c-446a-88cc-8a117bc9a9d7.PNG)

**Comment by erikrichardlarson** [comment]
Yo sorry you're having issues, is your KUVO playlist public?

**Comment by LuigiMIX** [comment]
yes, my song list is public, change configuration, thinking that was the problem, but to date unbox does not work

**Comment by erikrichardlarson** [comment]
Want to email me your KUVO user id so I can test on my end? You can email me at erikrichardlarson@gmail.com

**Comment by LuigiMIX** [comment]
I sent you bye Email, Kuvo User. thx

**Comment by erikrichardlarson** [comment]
Interesting I just tested your user id, and it worked as expected on Mac. Is it still not working on your end?

<img width="887" alt="Screen Shot 2022-09-12 at 9 58 27 PM" src="https://user-images.githubusercontent.com/17460309/189812417-58354363-7d1a-4605-8ee7-af069d457a71.png">

**Comment by LuigiMIX** [comment]
No, what are your next steps?

**Comment by erikrichardlarson** [comment]
Oh are you able to use the rekordbox real-time mode?

**Comment by LuigiMIX** [comment]
I'm sorry, I don't like that method, I'll look for another method, unbox doesn't work on windows. I'm looking for the solution elsewhere.. thx

**Comment by erikrichardlarson** [comment]
Gotcha, will let you know when I update the Windows version and KUVO is working again.

**Comment by LuigiMIX** [comment]
Thank you

**Comment by erikrichardlarson** [comment]
It looks like KUVO no longer supports public playlists unfortunately: https://kuvo.com/

For the real-time rekordbox mode, are you concerned about the memory edits?

**Comment by LuigiMIX** [comment]
the only functional method in real time? kuvo stopped working from before.

**Comment by erikrichardlarson** [comment]
Yeah it's the only real-time mode for Rekordbox. And yeah it's possible there were other issues before, but seems that KUVO is no longer available either way unfortunately.

**Comment by erikrichardlarson** [comment]
@LuigiMIX It looks like the latest rekordbox version allows you to configure how quickly the history file is written to so you should be able to use the standard rekordbox mode now, just let me know if you run into any issues.

**Comment by LuigiMIX** [comment]
hello! I don't understand your text. I should configure in Real-time mode? maybe my problem is recordbox, I have version 5.8.7, DDJ400 interface

**Comment by erikrichardlarson** [comment]
Ah gotcha, are you able to update your rekordbox version?

**Comment by LuigiMIX** [comment]
hi bro. Tutorial, for rekodbox 6.7, with the new unbox

**Comment by LuigiMIX** [comment]
"We're having trouble connecting. Please reach out to us for support." rekordbox 6.7 help!
